PyDSS/get_snapshot_timepoints.py

```diff
@@ -0,0 +1,57 @@
+"""Logic to determine snapshot time point by mode"""
+
+import logging
+
+from PyDSS.utils.simulation_utils import create_loadshape_pmult_dataframe_for_simulation
+from PyDSS.reports.reports import logger
+
+import pandas as pd
+import opendssdirect as dss
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_snapshot_timepoint(options):
+    pv_systems = dss.PVsystems.AllNames()
+    if not pv_systems:
+        logger.info("No PVSystems are present")
```

Comment:
@KwamiS @ann-sherin What timestamp should be returned if there are no PVSystems?

Reply:
If a feeder does not have PVSystems and user has chosen Max PV to Load ratio, we can log that there are no PV systems. And instead return the Max Load because that will be the next critical case. 

Side note: Currently, the code returns 'Max PV to Load ratio'. 
However timepoints_df has multiple columns (like timepoints for max load etc.) 
Can this be a parameter that the user defines? 

Or another option is to return Max PV to load ratio  timepoint by default? But if needed, user can choose to run for the other timepoints.